README.md

```diff
@@ -1,9 +1,71 @@
 # ensure-regression-tests
 
-Ensure Pull Requests have regression tests included - shell script that runs in your Travis build
+Add `ensure-regression-tests` to your Travis build to run tests *twice* when
+you create a pull request: once on the branch you’ve been working on, and once
+on the branch you’re targeting (commonly, this will be `master`). The tests on
+your branch *must pass* (of course)... but tests on the target branch *must
+fail*.
+
+Yes, Travis will reject your pull request if the **tests pass** on the target
+branch.
+
+There’s one useful exception: see the details below.
+
+`ensure-regression-tests` is implemented as a shell script, which you simply
+instruct Travis to use. You don’t need to install anything.
 
 ## Usage
 
+
 Add the following as a custom [Travis build step](https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Build-Step).
 
     bash <(curl -fsSL https://github.com/everypolitician/ensure-regression-tests/raw/v0.1.0/ensure-regression-tests)
+
+This will run v0.1.0 of the script (see [all versions available](https://github.com/everypolitician/ensure-regression-tests/releases)).
+
+If you want to use a specific version of the script, or perhaps always want the
+latest one, be explicit in the URL you supply to `curl`. That is, you can use
+the URL to a specific git commit SHA1 or tag within this
+`ensure-regression-tests` repo. For example, to execute the most recent version
+from the `master` branch (rather than being locked to a specific commit or release):
+
+    bash <(curl -fsSL https://github.com/everypolitician/ensure-regression-tests/raw/master/ensure-regression-tests)
+
+You can see this in use in
+[this live example](https://github.com/everypolitician/viewer-sinatra/blob/6058146aa548cde2f7db9cc98ed564e01577e8ff/.travis.yml#L13).
+
+## Details
+
+Broadly speaking there are three situations where this plays out:
+
+### New feature branch
+
+If you’re adding a new feature, you’d expect to add new tests for that new
+feature. You’d also expect those to fail because by definition the feature your
+tests are testing isn’t on the target branch (yet) — if it is, you’re not
+adding a new feature, you’re refactoring a new one (see below).
```

Comment:
"refactoring a new one" is slightly odd terminology here. I suspect this was just a thinko on "new".